packages/sdk/typescript/human-protocol-sdk/src/escrow.ts

```diff
@@ -937,6 +934,8 @@ export class EscrowClient extends BaseEthersClient {
     amounts: bigint[],
     finalResultsUrl: string,
     finalResultsHash: string,
+    txId: number,
```

Comment:
This new param is missing in tsdoc for both methods

Reply:
Fixed https://github.com/humanprotocol/human-protocol/pull/3012/commits/e901c12acc0166613bb863e558d7e0e4dd19c982